src/sas/qtgui/Plotting/PlotterBase.py

```diff
@@ -255,6 +260,42 @@ def onResize(self, event):
         """
         pass
 
+    def _zoom_pan_handler(self, event):
+        """
+        Catch zoom and pan events for graph range
+        """
+        pass
+
+    def _home(self, event):
+        """
+        Catch home button click events
+        """
+        pass
+
+    def _back(self, event):
+        """
+        Catch back button click events
+        """
+        pass
```

Comment:
Shouldn't the base method just fire the corresponding MPL toolbar event instead of passing?
e.g. `self.toolbar.back()`
Or alternatively, should it require implementation in derived class by adding a `NonImplemented` exception?